tools/devtools/src/page_scripts/owl_devtools_global_hook.js

```diff
@@ -1433,8 +1443,13 @@
           return;
         }
       }
-      const key = path.pop().value;
+      let key = path.pop();
```

Comment:
now, it's not really a key, is it? i propose `const elem = ...`
and then below:
```js
const key = elem.SymbolIndex ? ... : elem.value;
```